src/settings/Settings.ts

```diff
@@ -18,6 +18,7 @@ export interface MediaDbPluginSettings {
 	openNoteInNewTab: boolean;
 	useDefaultFrontMatter: boolean;
 	enableTemplaterIntegration: boolean;
+	embedPosters: boolean;
```

Comment:
I disagree with the name. Something like `generateEmbedLinkForImages` would be better